tools/cache_buster.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: denier
+metadata:
+  name: denyall
+spec:
+  status:
+    code: 7
+    message: Not allowed
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: checknothing
+metadata:
+  name: denyrequest
+spec:
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: mixercachebuster
+spec:
+  # one direction 1->2 won't be cached, 2->1 will use the cache
+  # TODO: parametrize the namespace of find a way to get short names to work:
+  # TODO: this appears to always bust the cache, even if dest is echosrv1 !
+  match: destination.service == "echosrv1.istio.svc.cluster.local" && request.headers["x-request-id"] == "foo"
```

Comment:
cc @mandarjog

Reply:
this shouldn't break the cache moving forward, correct?  have we verified with latest proxy (i believe there was an issue where we were still using a weeks-old build) ?